app/res/layout/loading.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/loadingContainer"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:contentDescription="@string/please_wait"
+    android:elevation="8dp"
+    android:gravity="center"
+    android:importantForAccessibility="noHideDescendants"
+    android:visibility="gone">
+
+    <ProgressBar
+        android:id="@+id/progressBar"
+        style="?android:attr/progressBarStyleLarge"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:gravity="center" />
```

Comment:
nit: don't think we need to specify gravity again given it's specified in parent